### DISCUSSION TITLE
Name of export file

### DISCUSSION THREAD

**Discussion by HappyException** [EvotecIT/PSWriteHTML#229] [General]
Is there a way to define the name of the file created when you export a table to Excel/CSV/PDF format?

**Comment by PrzemyslawKlys** [comment]
The name of the file comes from Title of new-HTML, if the title for New-HTMLTable is defined it comes from there, if it's defined per button - it's used from there. 

```powershell
        New-HTMLTable -DataTable $Test -PagingOptions @(50, 100, 150, 200) {
            New-TableButtonPDF -PageSize TABLOID -Orientation portrait
            New-TableButtonExcel -Title 'Custom'
        } -Title 'Or this one'
```

What's important to know that if you define buttons the way I show above and not define title the title from New-HTML will be used. You should explicitly define the title if you use New-TableButton...

I think I'll fix this inconsistency, as I would expect that New-HTMLTable title is used when New-TableButtonPDF doesn't set title explicitly.

**Reply by PrzemyslawKlys** [reply]
I've fixed this inconsistency. It will be available in the next release. Currently, live in pre-release but I would recommend against using pre-releases.
